plugins/inputs/ravendb/testdata/collections_full.json

```diff
@@ -0,0 +1,15 @@
+{
+  "PublicServerUrl": "http://localhost:8080",
+  "NodeTag": "A",
+  "Results": [
+    {
+      "DatabaseName": "db1",
+      "CollectionName": "Orders",
+      "DocumentsCount": 830,
+      "TotalSizeInBytes": 2744320,
+      "DocumentsSizeInBytes": 868352,
+      "TombstonesSizeInBytes": 122880,
+      "RevisionsSizeInBytes": 1753088
+    }
+  ]
+}
```

Comment:
Newline at end of file? Same for all other jsons..

Reply:
I don't have newlines at the end of json files and I assume it is correct, no ?

Reply:
Well it's a bit cumbersome if you for example just `cat` them and your prompt doesn't move on. Nothing big, but it would be nice to add that extra newline. :-)

Reply:
ok I get the point - thanks

Reply:
Thx! Please mark as resolved.